internal/cypress/docker/runner.go

```diff
@@ -234,15 +251,20 @@ func (r *Runner) teardown(logDir string) error {
 }
 
 func (r *Runner) runSuite(suite cypress.Suite) error {
+	var err error
+
 	defer func() {
-		log.Info().Msg("Tearing down environment")
+		if err == nil {
+			log.Info().Msg("Tearing down environment")
+		}
 		if err := r.teardown(r.Cli.LogDir); err != nil {
 			log.Error().Err(err).Msg("Failed to tear down environment")
 		}
 	}()
 
 	log.Info().Msg("Setting up test environment")
-	if err := r.setup(); err != nil {
+	if err = r.setup(); err != nil {
+		log.Error().Err(err).Msg("Failed to setup test environment")
```

Comment:
You can simplify this to 
```suggestion
		log.Err(err).Msg("Failed to setup test environment")
```

Reply:
Thanks!